Assets/Src/Scripts/tests.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021-2022 The SeedV Lab.
```

Comment:
1. As for the filename of this source code, please make sure the filename is capitalized in the same way as the main class name of the code. E.g., this code file should be named `Tests.cs`.

2. This code does nothing at all. Please do not include this kind of empty code in formal PRs.